### PR TITLE
update: add 1.6.20-RC EAP details

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**

### CODE REVIEW

**Comment by PetrakovichVictoria** [comment]
Screenshot 2022-02-17 at 17 57 15

File: docs/topics/eap.md [hunk 1]
```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
...
+        <td><strong>1.6.20-RC</strong>
+            <p>Released: <strong>February 21, 2022</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-RC" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <p>Stabilization of Kotlin 1.6.20-M1 features.</p>
+            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-RC">changelog</a>.</p>
+        </td>
+        <td>
+            <p>The same as for the Kotlin 1.6.20-M1, see the row below.</p>
```
Comment (p7nov) [inline docs/topics/eap.md:45]: Let's maybe just merge cells?

**Review by p7nov — APPROVED** [review]
LGTM; one suggestion

File: docs/topics/eap.md (hunk 1 above)
Comment (p7nov) [reply docs/topics/eap.md:45]: Ah, this looks really awkward with our table style. Forget it.
Just remove the first two "the" articles in your sentence.

Comment (PetrakovichVictoria) [inline docs/topics/eap.md:45]:
```suggestion
            <p>Same as for Kotlin 1.6.20-M1, see the row below.</p>
```

File: docs/topics/eap.md [hunk 2]
```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.6.20-RC</strong>
+            <p>Released: <strong>February 21, 2022</strong></p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:37]:
```suggestion
            <p>Released: <strong>February 22, 2022</strong></p>
```

File: docs/topics/eap.md [hunk 3]
```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.6.20-RC</strong>
+            <p>Released: <strong>February 22, 2022</strong></p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:37]:
```suggestion
            <p>Released: <strong>February 28, 2022</strong></p>
```

File: docs/topics/eap.md [hunk 4]
```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.6.20-RC</strong>
+            <p>Released: <strong>February 28, 2022</strong></p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:37]:
```suggestion
            <p>Released: <strong>March 1, 2022</strong></p>
```

**Pull request merged**